for/main.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
Thank you for the comment!!!
I got the reason why because too many main.go to run?
```
go: pattern ./... matches multiple packages:
        github.com/snkzt/go-practises/arrays
        github.com/snkzt/go-practises/for
        github.com/snkzt/go-practises/ifElse
        github.com/snkzt/go-practises/map
        github.com/snkzt/go-practises/slices
        github.com/snkzt/go-practises/switch
```

Reply:
The reason is that you can't write `package for` or `package switch` since `for` and `switch` are reserved keywords in Go. The compiler wouldn't understand what you meant.

You can't do `go run ./...` when you have multiple `main()` functions since Go doesn't know which one to run. So yes, but I think you answered a different question than what I asked 🙂

Reply:
Very clear now!

Reply:
You cannot have several main.go in a directory.